docs/topics/assembly_developing-and-deploying-spring-reactive-application.adoc

```diff
@@ -0,0 +1,24 @@
+[id='developing-reactive-applications-using-spring-boot-vertx_{context}']
+= Developing reactive applications using {SpringReactive}
+
+:parent-context: {context}
+
+The following sections provide a conceptual introduction to functional principles behind {ProductShortName} reactive application programming with Spring, as well as basic examples of implementing reactive application design patterns using the {SpringReactive} starters.
```

Comment:
`to functional principles behind Application Development on OpenShift reactive application programming with Spring` somehow just doesn't seem correct to me

Reply:
You're right. Sounds weird... I  made the introduction simpler and rephrased the sentence completely.

Reply:
:+1: